dspace-api/src/main/java/org/dspace/authorize/dao/ResourcePolicyDAO.java

```diff
@@ -79,4 +80,33 @@
      */
     public List<ResourcePolicy> findByDSoAndActionExceptRpType(Context c, DSpaceObject o, int actionID,
             String rpType) throws SQLException;
+
+    public List<ResourcePolicy> findbyEPerson(Context c, EPerson ePerson, int offset, int limit) throws SQLException;
+
+    public int searchCountEPerson(Context context, EPerson eperson) throws SQLException;
```

Comment:
Same here, `searchCount` could be renamed to `count` or `countBy` (throughout this class)